renku/cli/run.py

```diff
@@ -15,7 +15,63 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Track provenance of data created by executing programs."""
+"""Track provenance of data created by executing programs.
+
+Capture command line execution
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Tracking exection of your command line script is done by simply adding
+``renku run`` command before the previous arguments. The command will detect
+
+* arguments (flags),
+* string and integer options,
+* input files if linked to existing files in the repository,
+* output files if modified or created while running the command.
+
+.. note:: If there were uncommitted changes then the command fails.
+   Check :program:`git status` to see details.
+
+Detecting input files
+~~~~~~~~~~~~~~~~~~~~~
+
+The arguments are identified as input file only if they path matches an
```

Comment:
An argument is identified as an input file only if its path matches an existing file in the repository.